client/landing/stepper/declarative-flow/newsletter.ts

```diff
@@ -35,6 +35,10 @@ const newsletter: Flow = {
 						{ slug: 'intro', asyncComponent: () => import( './internals/steps-repository/intro' ) },
 				  ]
 				: [] ),
+			{
+				slug: 'newsletterType',
+				asyncComponent: () => import( './internals/steps-repository/newsletter-type' ),
+			},
```

Comment:
We should probably duplicate this whole flow definition list and put new one behind a feature flag.

The feature can also hide the checkbox from setup step.

Reply:
In subsequent conversations on Slack, I think we decided to approach this is a more incremental way, ideally with each step being pushed to production. And I think we have a good approach for doing that. If not, we can always come back to separate flow + feature flag.